packages/server/graphql/intranetSchema/mutations/updateWatchlist.ts

```diff
@@ -0,0 +1,49 @@
+import {GraphQLBoolean, GraphQLID, GraphQLNonNull} from 'graphql'
+import getRethink from '../../../database/rethinkDriver'
+import updateUser from '../../../postgres/queries/updateUser'
+import {requireSU} from '../../../utils/authorization'
+
+const updateWatchlist = {
+  type: GraphQLBoolean,
+  description: 'add a user to the LogRocket watchlist so that their sessions are recorded',
+  args: {
+    userId: {
```

Comment:
maybe accept emails instead of userId? we might want to log entire domains, too (in the case of symph).

API might end up looking like: https://github.com/ParabolInc/parabol/blob/f377b7f9ba79fa11174ad0a6ab7037e6c6a1b3d6/packages/server/graphql/mutations/addFeatureFlag.ts

Reply:
Thanks, that makes sense! I'm popping out to dinner now so will merge this so that it's in the ship. Then I can refactor this tomorrow so we can use domains and emails in the future